lib/qmp.mli

```diff
@@ -35,6 +35,8 @@ type fd_info = {
   fdset_id : int;
 }
 
+type qom = {name:string; ty:string;}
```

Comment:
Please, use the same style as above and add a small docstring explaining what does it represent.

Reply:
updated